.clj-kondo/manifold/hooks.clj

```diff
@@ -0,0 +1,100 @@
+(ns manifold.hooks
+  (:require [clj-kondo.hooks-api :as api]))
+
+(defn- cons-vector-node
+  [node parent]
+  (api/vector-node (cons node (:children parent))))
```

Comment:
rewrite-clj has `children` and `replace-children` for this sort of thing, but I don't know if kondo's altered version of rewrite-clj does (or exposes them publicly)

Reply:
As far as I know the documentation doesn't mention `replace-children`, it also doesn't show up in the `keys` of vector nodes.